robotics_controller/Trajectory goal_points

--- 

bool success 
